.github/workflows/build_and_test.yml

```diff
@@ -202,45 +219,48 @@ jobs:
           docker logs opentelemetry-collector
 
     - name: Install AWS SAM cli tool
-      if: ${{ matrix.suite == 'plugins' }}
       run: |
           curl -L -s -o /tmp/aws-sam-cli.zip https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip
           unzip -o /tmp/aws-sam-cli.zip -d /tmp/aws-sam-cli
           sudo /tmp/aws-sam-cli/install --update
 
-    - name: Update PATH
-      run: |
-        echo "$BUILD_ROOT/kong-dev/bin" >> $GITHUB_PATH
-        echo "$BUILD_ROOT/kong-dev/openresty/nginx/sbin" >> $GITHUB_PATH
-        echo "$BUILD_ROOT/kong-dev/openresty/bin" >> $GITHUB_PATH
-
-    - name: Debug (nginx)
-      run: |
-        echo nginx: $(which nginx)
-        nginx -V 2>&1 | sed -re 's/ --/\n--/g'
-        ldd $(which nginx)
-
-    - name: Debug (luarocks)
+    - name: Create kong_ro user in Postgres
       run: |
-        echo luarocks: $(which luarocks)
-        luarocks --version
-        luarocks config
+        psql -v ON_ERROR_STOP=1 -h localhost --username kong <<\EOD
+                CREATE user kong_ro;
+                GRANT CONNECT ON DATABASE kong TO kong_ro;
+                \c kong;
+                GRANT USAGE ON SCHEMA public TO kong_ro;
+                ALTER DEFAULT PRIVILEGES FOR ROLE kong IN SCHEMA public GRANT SELECT ON TABLES TO kong_ro;
+        EOD
 
     - name: Tune up postgres max_connections
       run: |
         # arm64 runners may use more connections due to more worker cores
         psql -hlocalhost -Ukong kong -tAc 'alter system set max_connections = 5000;'
 
-    - name: Generate test rerun filename
-      run: |
-           echo FAILED_TEST_FILES_FILE=$(echo '${{ github.run_id }}-${{ matrix.suite }}-${{ matrix.split }}' | tr A-Z a-z | sed -Ee 's/[^a-z0-9]+/-/g').txt >> $GITHUB_ENV
+    - name: Download test schedule file
+      uses: actions/download-artifact@v3
+      continue-on-error: true
+      with:
+        name: schedule-test-files
 
+    - name: Generate helper environment variables
+      run: |
+           echo FAILED_TEST_FILES_FILE=failed-tests.txt >> $GITHUB_ENV
+           echo TEST_FILE_RUNTIME_FILE=test-runtime.txt >> $GITHUB_ENV
 
     - name: Download test rerun information
       uses: actions/download-artifact@v3
       continue-on-error: true
       with:
-        name: ${{ env.FAILED_TEST_FILES_FILE }}
+        name: test-rerun-info-${{ matrix.runner }}
```

Comment:
Just curious with this change, is it still possible to rerun all of the tests (including both failed and succeeded) from Github? I'm asking this question as I'm building a flakiness KPI where I need to keep rerun all of the test cases for certain time regardless if they failed or not.

Reply:
Yes, you can just re-run all workflows, which will run the scheduling process and all workers.